cerberus/validator.py

```diff
@@ -666,8 +666,8 @@ def __normalize_sequence(self, field, mapping, schema):
             document_crumb=field, schema_crumb=(field, 'schema'),
             schema=schema)
         result = validator.normalized(document, always_return_document=True)
-        for i in result:
-            mapping[field][i] = result[i]
+        if result:
```

Comment:
with my previous comment i meant two things, sorry for being unclear:
- this expression is semantically identical to the previous version
- the expression shouldn't be needed at all, bit i might be wrong with that assumption